src/app/public/modules/repeater/repeater-item.component.ts

```diff
@@ -71,12 +74,24 @@ let nextContentId: number = 0;
 })
 export class SkyRepeaterItemComponent implements OnDestroy, OnInit, AfterViewInit {
 
+  /**
+   * Specifies configuration options for the buttons to display on an inline form
+   * within the repeater. This property accepts a `SkyInlineFormConfig` object.
+   */
   @Input()
   public inlineFormConfig: SkyInlineFormConfig;
 
+  /**
+   * Specifies a template to use to instantiate an inline form within the repeater.
+   * This property accepts [an Angular TemplateRef class](https://angular.io/api/core/TemplateRef).
+   */
   @Input()
   public inlineFormTemplate: TemplateRef<any>;
 
+  /**
+   * Indicates whether the repeater item is expanded.
+   * @default true
```

Comment:
In the existing docs, the default is marked as `false`. Looking at the code here, that looks like a mistake. But just to make sure, `true` is the correct default, right?

Reply:
Yep that was a total mistake. I corrected it here.